tasks/nexus_install.yml

```diff
@@ -61,7 +77,8 @@
       - "{{ nexus_installation_dir }}/nexus-latest/etc/jetty"
       - "{{ nexus_installation_dir }}/nexus-latest/etc/fabric"
       - "{{ nexus_installation_dir }}/nexus-latest/etc/logback"
-  when: nexus_version >= '3.1.0'
+      - "{{ nexus_installation_dir }}/nexus-latest/etc/scripts"
+  when: nexus_version >= '3.2.1-01'
```

Comment:
I think this line should stay as it was `nexus_version >= '3.1.0'` ? there was a change in the config paths at version 3.1.0, so there is no need to bump this version (see line 66 above).